src/main/java/org/stellar/sdk/responses/operations/ExtendFootprintTTLOperationResponse.java

```diff
@@ -3,7 +3,9 @@
 import com.google.gson.annotations.SerializedName;
 
 /**
- * Represents BumpFootprintExpiration operation response.
+ * Represents ExtendFootprintTTL operation response.
+ *
+ * <p>TODO: update link
```

Comment:
Currently, the code for Horizon has not been updated, so we need to update this link later.

Reply:
ok, https://github.com/stellar/go/issues/5089 is pending for update on horizon